src/handlers/comment/handlers/index.ts

```diff
@@ -176,20 +177,24 @@ export const issueReopenedCallback = async (): Promise<void> => {
     }
     const assignee = events[0].assignee.login;
 
-    // write penalty to db
-    try {
-      await addPenalty(assignee, repository.full_name, tokenAddress, networkId.toString(), amount);
-    } catch (err) {
-      logger.error(`Error writing penalty to db: ${err}`);
-      return;
-    }
+    if (formattedAmount != "0.0") {
```

Comment:
I wonder if this is the most robust way to handle this. I don't have any other suggestions at this time.

Reply:
I'll parse it as a float check for ``> 0``